Public/New-ChartDataLabel.ps1

```diff
@@ -0,0 +1,30 @@
+﻿function New-ChartDataLabel {
+    [cmdletBinding()]
+    param(
+        [switch] $Enabled,
+        [int] $DataLabelsOffsetX,
+        [string] $DataLabelsFontSize,
+        [string[]] $DataLabelsColor
+    )
+
+    $Object = [PSCustomObject] @{
+        ObjectType = 'DataLabel'
+        DataLabel  = [ordered] @{
+            enabled = $Enabled.IsPresent
```

Comment:
This will always make it enabled.

-Enabled 
-Enabled:$true
-Enabled:$false 

Will all validate `$Enabled.IsPresent` to true.

You should just do:
`enabled = $Enabled`

Reply:
Mmm, no it won't, as far as my testing goes.

![image](https://user-images.githubusercontent.com/15063294/99505541-3fa17d00-2981-11eb-9638-a308fa130155.png)

```powershell
            New-HTMLChart -Title 'Days' -TitleAlignment center {
                New-ChartAxisY # this hides AXIS Y
                New-ChartDataLabel -Enabled:$false -DataLabelsColor Black, Red, Red, Red, Red
                $From = (Get-Date).AddDays(-1)
                $To = (Get-Date).AddDays(2)
                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Analysis' -Color Red
                $From = (Get-Date).AddDays(0)
                $To = (Get-Date).AddDays(5)
                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Design' -Color Yellow
                $From = (Get-Date).AddDays(1)
                $To = (Get-Date).AddDays(8)
                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Coding' -Color Beige
                $From = (Get-Date).AddDays(3)
                $To = (Get-Date).AddDays(7)
                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Testing' -Color Beige
                $From = (Get-Date).AddDays(8)
                $To = (Get-Date).AddDays(10)
                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Deployment' -Color Beige
            }
```

The problem with leaving it as $Enabled (which is easier to work with) is that when it's converted to JSON it creates something else 

![image](https://user-images.githubusercontent.com/15063294/99505688-6cee2b00-2981-11eb-8414-0702f0abe31c.png)

Reply:
I stand corrected. I had some guy show me way back in 5.1, that IsPresent was going to give me false results. Let me see if I can recreate it...